prow/config/external_plugins_config.yaml

```diff
@@ -1353,6 +1357,8 @@ ti-community-tars:
       - pingcap/tiflash
       - pingcap/tiflash-scripts
       - pingcap/tispark
+      - pingcap/tiunimanager
```

Comment:
The `pingcap/tiunimanager` didn't use the Jenkins CI, so we don't need to `/run-all-tests` after the bot performed the merge-base operation.